src/chainparamsbase.h

```diff
@@ -22,15 +22,13 @@ class CBaseChainParams
 public:
     const std::string& DataDir() const { return strDataDir; }
     uint16_t RPCPort() const { return m_rpc_port; }
-    uint16_t OnionServiceTargetPort() const { return m_onion_service_target_port; }
```

Comment:
i think we could have kept this, and made it return `port + 1`? It would have made the overall patch smaller.

Oh wait, no, we don't have the normal P2P port information here. Never mind.